plugins/drm/providers/fairplay/FairplayPlugin.php

```diff
@@ -171,4 +171,26 @@ private static function shouldEditManifest($config)
 		return false;
 	}
 
+	public function contributeToEntryPlayingDataResult(entry $entry, KalturaEntryPlayingDataParams $entryPlayingDataParams, KalturaEntryPlayingDataResult $result)
+	{
+		if ($this->shouldContribute($entry) && $this->isSupportStreamerTypes($entryPlayingDataParams->deliverProfile->getStreamerType()))
+		{
+			$data = new KalturaFairPlayEntryPlayingPluginData();
+			$data->scheme = $this->getPluginName();
```

Comment:
This isn't good, the plugin name is "fairplay" but the scheme should be "fps"
I think maybe in general you should have a "getSchemeName" method in each drm plugin